konduit-serving-pipeline/src/main/java/ai/konduit/serving/pipeline/api/context/Profiler.java

```diff
@@ -65,12 +67,12 @@
     /**
      * Start the timer for the event with the specified key.
      */
-    void eventStart(String key);
+    void eventStart(String key) throws IOException;
 
     /**
      * End the timer for the event with the specified key.
      * Should be called some time after {@link #eventStart(String)}
      */
-    void eventEnd(String key);
+    void eventEnd(String key) throws IOException;
```

Comment:
Remove these checked exceptions 
We don't want users/devs to have to add a try/catch every single time they call a profiler method.